<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugExpert</class>
 <widget class="QWidget" name="DebugExpert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_freeze_data">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>参数未固化</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_multi_server">
         <property name="text">
          <string>主机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_multi_client1">
         <property name="text">
          <string>客1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_multi_client2">
         <property name="text">
          <string>客2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_multi_close">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_sync">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="TTKRadioButtonWidget" name="eth_on">
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKRadioButtonWidget" name="eth_off">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lab_multi">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>多机互联</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lab_freeze_data">
       <property name="text">
        <string>固化参数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lab_temp">
       <property name="text">
        <string>温度补偿</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lab_sync">
       <property name="text">
        <string>同步器匹配</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lab_eth">
       <property name="text">
        <string>以太网口</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_auto">
         <property name="text">
          <string>智能</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_uncharge">
         <property name="text">
          <string>未充电</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKRadioButtonWidget" name="rbt_charge">
         <property name="text">
          <string>充电</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTKRadioButtonWidget</class>
   <extends>QRadioButton</extends>
   <header location="global">Gui/TTK/ttkradiobuttonwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
